pandas/tests/plotting/test_boxplot_method.py

```diff
@@ -740,3 +740,17 @@ def test_boxplot_multiindex_column(self):
         expected_xticklabel = ["(bar, one)", "(bar, two)"]
         result_xticklabel = [x.get_text() for x in axes.get_xticklabels()]
         assert expected_xticklabel == result_xticklabel
+
+    @pytest.mark.parametrize("group", ["X", ["X", "Y"]])
+    def test_boxplot_multi_groupby_groups(self, group):
+        # GH 14701
+        rows = 20
+        df = DataFrame(
+            np.random.default_rng(12).normal(size=(rows, 2)), columns=["Col1", "Col2"]
+        )
+        df["X"] = Series(np.repeat(["A", "B"], int(rows / 2)))
+        df["Y"] = Series(np.tile(["C", "D"], int(rows / 2)))
+        pregrouped = df.groupby(group)
```

Comment:
nit: Just to adhere to existing conventions in tests, could you either call this `grouped` or `gb`. These are the names used in other tests here.